src/022/p022.red

```diff
@@ -0,0 +1,15 @@
+Red [Title: "Problem 022"]
+
+names: sort split read %p022_names.txt #","
+
+score: function [idx name][
+  s: 0 
+  foreach n do name [s: s + (to integer! n) - 64]
+  idx * s
+]
+
+scores: 0
+
+forall names [scores: scores + score (index? names) (first names)]
+
+print scores
```

Comment:
Maybe you could wrap these last lines in a "solve" function.
What do you think?